source/javascripts/application.js

```diff
@@ -15,7 +15,7 @@
         $('.js-full-height').height(viewport - 30);
     });
 
-    $('a').click(function(){
+    $('nav-side a').click(function(){
```

Comment:
The selector was too greedy and was causing exceptions while navigating away from pages. As far as I can tell you only want this to happen when you click the little nav circles, so I refined the selector.